### ISSUE TITLE
杨博士您好，我对MPED.py代码里有一处代码有疑问，想求证下

### ISSUE THREAD

**Issue by linsanityyong** [Qi-Yangsjtu/MPED#1]
在求SPED时，您直接将每个center point在两个点云中K近邻的能量对应相减取绝对值，再求和得到每个center  point的能量差，原文里不应该是将每个center point在两个点云中K近邻的能量内部求和后，再相减取绝对值嘛？
 dis_energy = (energy_x - energy_y).abs()  # 50,2048,3
 dis_energy = dis_energy.sum(dim=2) #
 dis_energy = dis_energy.sum(dim=1)
我认为正确的做法：
energy_x = torch.sum(energy_x, dim=2)#先将每个点的能量计算出来
energy_y = torch.sum(energy_y, dim=2)
dis_energy = (energy_x - energy_y).abs()#计算每个点在两个点云中的能量差
dis_energy =  dis_energy.sum(dim=1)#计算所有点的能量差和#torch.size(N)

**Comment by Qi-Yangsjtu** [comment]
感谢提醒，这个地方的错误我前段时间发现了，但是因为忙着毕业一直没有来得及更新，刚才把code都更新了一下，希望对你有所帮助。
